### PR TITLE
239 format backendbackend dir

### PR DESCRIPTION
**Pull request by sakarisalmi**
Tidying up different parts of the backend to get the project evaluation ready. Expect similar PRs of different backend apps soon.

### CODE REVIEW

**Review by azarSarikhani — APPROVED** [review]

File: backend/backend/urls.py [hunk 1]
```diff
@@ -2,89 +2,106 @@
...
-	path('tournaments/<int:tournament_id>/matches/', TournamentMatchListView.as_view()),
-	path('tournaments/<int:tournament_id>/matches/<int:tournament_match_id>', TournamentMatchDetailView.as_view()),
-
-	# Stats Views
-	path('stats/', StatsView.as_view()),
-	path('leaderboard/', LeaderBoardView.as_view()),
+    path('admin/', admin.site.urls),
+
+    # User views
+    path('users/', UserListView.as_view(), name='user-list'),
```
Comment (merituulie) [inline backend/backend/urls.py:73]: the name=user-list would be for use IF we used templates but as we dont, Im not sure we should add them here. Checking the documentation might help. But if it doesnt break anything than fine..

Comment (merituulie) [reply backend/backend/urls.py:73]: just linking this issue here as itll be done in this one too, nice! https://github.com/dorianjagusch/ft_transcendence/issues/237

Comment (dorianjagusch) [reply backend/backend/urls.py:73]: Will deal with it. Name is indeed not necessary

**Comment by dorianjagusch** [comment]
Would be best,, if @sakarisalmi removed the name tags (tbf they don't hurt here either)

File: backend/backend/urls.py (hunk 1 above)
Comment (sakarisalmi) [reply backend/backend/urls.py]: I removed the name stuff!

**Pull request closed without merging**